recipe/meta.yaml

```
@@ -1,42 +1,40 @@
 {% set name = "twine" %}
-{% set version = "6.0.1" %}
+{% set version = "6.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/twine-{{ version }}.tar.gz
-  sha256: 36158b09df5406e1c9c1fb8edb24fc2be387709443e7376689b938531582ee27
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e5ed0d2fd70c9959770dce51c8f39c8945c574e18173a7b81802dab51b4b75cf
 
 build:
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
-  number: 0
-  skip: true  # [py<38]
+  skip: true  # [py<39]
   entry_points:
     - twine = twine.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
 
 requirements:
   host:
     - python
     - pip
     - wheel
-    - setuptools >=61.2
+    - setuptools >=77.0.3
     - setuptools_scm >=6.0
```

https://github.com/pypa/twine/blob/6.2.0/pyproject.toml#L3
```suggestion
    - setuptools_scm >=6.0
    - toml
```